Marlin/pins_GEN6.h

```diff
@@ -84,3 +111,15 @@
 // RS485 pins
 #define TX_ENABLE_PIN      12
 #define RX_ENABLE_PIN      13
+
+// added 26 DEC 2016
+
+#if ENABLED(SPINDLE_ENABLE)
+  #define SPINDLE_ENABLE_PIN 5      // should have a pull up/down on it
+  #define SPINDLE_SPEED_PIN  16     // MUST BE A HARDWARE PWM
+  #define SPINDLE_DIR_PIN    6
+#endif
```

Comment:
You should remove the `#if` blocks on these pins and the case light pin below. In Marlin we want to always define the pins for every feature, regardless of whether the feature is enabled or not, so we don't have to have tests like this in every pins file. (It makes them become less manageable.) If there is some conflict between the spindle and some other feature then it should be flagged in `SanityCheck.h`.